platform/src/main/resources/config/anchor-config-default-values.yaml

```diff
@@ -145,27 +145,43 @@ event_processor:
   callback_api_request:
     enabled: true
 
-# The list of clients for the Anchor server to safely communicate with the outside client domains.
+# The list of clients for the Anchor server to safely communicate with the outside wallet servers or clients.
 # Each item in the list may contain the following fields:
-#   - domain: the domain of the client
-#   - type: the type of the client. The supported values are `custodial` and `non-custodial`
-#   - public_key: the public key of the client. This must match the public key of the TOML file of the client domain
-#   - url: the URL of the client's callback API endpoint
+#   - type: `custodial` or `noncustodial`
+#
+#   If the type is `custodial`,
+#   - account: (required) the custodial account of the client. If the `account` of the transaction is the same as this
+#              account, the callback will be activated. The `account` field of the transaction is the account that is
+#              used for the SEP-10 authentication.
+#   - callback_url: (required) the URL of the client's callback API endpoint
+#
+#   If the type is `noncustodial`,
+#   - domain: (required) the domain of the client. If the `client_domain` field of the transaction is the same as this
+#             domain, the callback will be activated.
+#   - callback_url: (required) the URL of the client's callback API endpoint
+#   - public_key: (optional) the public key of the client. If the public key is specified, the client domain's TOML
```

Comment:
Is there a reason we need a public key in config, if we can always get it from TOML?

Reply:
It is for verification process. If not specified, it won't be verified.